packages/demo-cards/claim.gts

```diff
@@ -17,6 +18,11 @@ import { on } from '@ember/modifier';
 import { action } from '@ember/object';
 
 class Isolated extends Component<typeof Claim> {
+  @tracked isClaimed = false;
+  // these is no good way to load types from a URL
```

Comment:
It would be ok to use something like `import type { whatever } from '@cardstack/cardpay-sdk'` and making sure that the dependency is available in the monorepo during development.

When you use `import type` it's guaranteed to have no runtime effect.